### ISSUE TITLE
Provide actual setup instructions

### ISSUE THREAD

**Issue by NathanSweet** [actions-on-google/smart-home-local#55]
This project makes too many assumptions that the user knows what they are doing. For example:

> Deploy the placeholder smart home provider to Cloud Functions for Firebase using the same Project ID: 

I have no idea what that means, I assume it's what the commands do. Except what folder am I supposed to run the listed commands? From the root of the project doesn't work:

```
npm install --prefix functions/
npm ERR! code ENOLOCAL
npm ERR! Could not install from "" as it does not contain a package.json file.
```

So I need to run the command where there is a `package.json`? The `app`, `device`, and `functions` folders have a `package.json`. Which do I use? OK the command succeeds in the `app` subfolder but the next command does not:

```
npm run firebase --prefix functions/ -- use ${PROJECT_ID}
npm ERR! code ENOENT
npm ERR! syscall open
npm ERR! path [snip]\app\functions\package.json
npm ERR! errno -4058
npm ERR! enoent ENOENT: no such file or directory, open '[snip]\app\functions\package.json'
```

I have no idea. Please provide instructions that can be followed to get the desired result.

**Comment by devedse** [comment]
I also ran into this error. I fixed this by running the commands from the functions/ directory themselves.
e.g.

```
cd functions
npm install
```

**Comment by bugs181** [comment]
Yup. Recently switched from a new iPhone 12 Pro to Google Pixel 6 Pro and migrating all of my home automation over. I am completely new to Google services and APIs. I've been trying to get this example running for over 5 hours.
